components/x11/libfontenc/libfontenc.p5m

```diff
@@ -24,7 +24,7 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 <transform file path=usr/lib(.*)/pkgconfig/(.+).pc$ -> set action.hash usr/lib/xorg%<\1>/pkgconfig/%<\2>.pc >
 
-depend fmri=x11/library/libfontenc/encodings type=require
+#depend fmri=x11/library/libfontenc/encodings type=require
```

Comment:
What's wrong in circular dep? It's not a build dep. IIRC, the idea is that if application require libfontenc (which is auto-detected), it automatically gets encodings, which are required for normal libfontenc work.